decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -225,6 +225,10 @@ def add_ignore_uploads
         append_file ".gitignore", "\n# Ignore public uploads\npublic/uploads" unless options["skip_git"]
       end
 
+      def add_ignore_tailwind_configuration
+        append_file ".gitignore", "\n\n# Ignore Tailwind configuration\ntailwind.config.js" unless options["skip_git"]
```

Comment:
I'd also include this addition to `.gitignore` in the changelog notes for "Tailwind CSS introduction". The same way as in the 0.27 changelog for service workers:
https://github.com/decidim/decidim/blob/release/0.27-stable/CHANGELOG.md#28-service-workers